.github/CHANGELOG.md

```
@@ -207,4 +207,28 @@ No changelog available.
 1. IR target.
 1. Runtime flags.
 ### Fixed
-1. Full issue coverage now.
\ No newline at end of file
+1. Full issue coverage now.
+
+## v8 -- Semantic solver
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)